include/oboe/AudioStream.h

```diff
@@ -242,14 +243,20 @@ class AudioStream : public AudioStreamBase {
      * and the sample format. For example, a 2 channel floating point stream will have
      * 2 * 4 = 8 bytes per frame.
      *
+     * Note for compressed formats, bytes per frames is treated as 1 by convention.
+     *
      * @return number of bytes in each audio frame.
      */
-    int32_t getBytesPerFrame() const { return mChannelCount * getBytesPerSample(); }
+    int32_t getBytesPerFrame() const {
+        return isCompressedFormat(mFormat) ? 1 : mChannelCount * getBytesPerSample(); }
```

Comment:
The messaging in getBytesPerFrame() and getBytesPerSample() aren't consistent. Should we return the same number for both?

Reply:
Good catch! For compressed format, we consider bytes per sample as 0 due to not fixed size per sample. For bytes per frame, it is considered as 1 by convention. I updated the comments.